kubeflow-katib.yaml

```diff
@@ -1,11 +1,14 @@
 package:
   name: kubeflow-katib
-  epoch: 13
+  epoch: 14
   version: 0.17.0
   description: Kubeflow Katib services
   copyright:
     - license: Apache-2.0
 
+vars:
+  py-version: 3.11
```

Comment:
Solution looks good to me. Any reason we've moved to 3.11 here though?

Reply:
I looked at the upstream and they had python 11 and thought it might be appropriate to do so for our packages